### PR TITLE
Feature/server setup

### PR DESCRIPTION
**Pull request by vivekimsit**
Todos
---

- [x] Write boilerplate code
- [x] Setup logging
- [x] Setup config
- [x] Setup linter

I feel that I can split the remaining tasks in future PRs otherwise this PR will become too big to be reviewed.

### CODE REVIEW

**Comment by nikrb** [comment]
so far so good :)

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "node": true,
+    "mocha": true
+  },
+  "rules": {
+    "no-console": 1
+  },
+  "plugins": [
+    "react"
+  ]
+}
```
Comment (Mortuie) [inline .eslintrc.json]: So have you implemented an eslint ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "genie",
+  "version": "1.0.0",
+  "description": "The ultimate solution to your housing issues",
```
Comment (Mortuie) [inline package.json]: Hehehehe :)

**Review by Mortuie — COMMENTED** [review]
Oh wow you are good at coding. One thing. Are we using 4 spaces or 2 spaces to indent. You have used two on the backend and we have used 4 on the frontend. We should decide for one and just go with it. I really don't like having two styles in one ecosystem. Especially when using one language. My two cents I prefer 4 spaces. But up to discussion.

File: package.json (hunk 2 above)
Comment (vivekimsit) [reply package.json]: Feeling too lazy to think of a name ;)

File: .eslintrc.json (hunk 1 above)
Comment (vivekimsit) [reply .eslintrc.json]: Yes, but it contains bare minimum rules although we can keep on adding more rules as needed. I haven't forced linting yet but we can discuss it. The commands for running lint can be found in server `README`

**Comment by vivekimsit** [comment]
Closes #10

**Comment by Mortuie** [comment]
See my comment about linting @vivekimsit

**Comment by Mortuie** [comment]
`Oh wow you are good at coding. One thing. Are we using 4 spaces or 2 spaces to indent. You have used two on the backend and we have used 4 on the frontend. We should decide for one and just go with it. I really don't like having two styles in one ecosystem. Especially when using one language. My two cents I prefer 4 spaces. But up to discussion.`

**Comment by vivekimsit** [comment]
@Mortuie check [Google Style Guide](https://google.github.io/styleguide/jsguide.html)

**Comment by Mortuie** [comment]
@vivekimsit So we are following a google linter/style guide ?

**Comment by vivekimsit** [comment]
@Mortuie ideally we should, anyways its just my opinion.

**Review by Mortuie — APPROVED** [review]

**Pull request merged**